model/src/main/kotlin/OutputFormat.kt

```diff
@@ -25,7 +25,7 @@ import com.fasterxml.jackson.module.kotlin.readValue
 import java.io.File
 
 /**
- * An enumeration of supported output file formats, their primary [fileExtension], and optional [aliases] (not including
+ * An enumeration of supported output file formats, their primary [fileExtension], and optional aliases (not including
```

Comment:
Any idea why it is not possible to reference `aliases`? Is it because it's not a property like the others?

Reply:
Exactly, it's only a constructor argument, so it could only be referenced in the KDoc of the constructor itself, not in the KDoc of the class.